README.md

```diff
@@ -84,9 +84,10 @@ To indicate which days are blocked from selection, you may provide a function to
   isDayBlocked: PropTypes.func
 ```
 
-If you would like to allow the user to selected days in the past, you may set `allowPastDates` to true.
+`isOutsideRange` indicates which days are out of selectable range.
+Past dates out of range by default. If you would like to allow the user to select days in the past, you may set `isOutsideRange` to `() => true`.
```

Comment:
Might be good to add a comment about the relationship between `isOutsideOfRange` and `doesNotMeetMinimumNights`. This is not blocking and I can add this after.